cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js

```diff
@@ -187,17 +189,17 @@ export const getProfiles = (namespace) => {
         const extraTags = {
           programtype: 'Workflow'
         };
-        let oneDayUSERMetricsBody = getProfileMetricsBody('oneDayUSERMetrics', namespace, 'USER', 'now-24h', 'now', extraTags);
-        let overAllUSERMetricsBody = getProfileMetricsBody('overAllUSERMetrics', namespace, 'USER', 0, 0, extraTags);
-        let oneDaySYSTEMMetricsBody = getProfileMetricsBody('oneDaySYSTEMMetrics', namespace, 'SYSTEM', 'now-24h', 'now', extraTags);
-        let overAllSYSTEMMetricsBody = getProfileMetricsBody('overAllSYSTEMMetrics', namespace, 'SYSTEM', 0, 0, extraTags);
+        let oneDayUSERMetricsBody = getProfileMetricsBody('oneDayUSERMetrics', namespace, SCOPES.USER, 'now-24h', 'now', extraTags);
+        let overAllUSERMetricsBody = getProfileMetricsBody('overAllUSERMetrics', namespace, SCOPES.USER, 0, 0, extraTags);
+        let oneDaySYSTEMMetricsBody = getProfileMetricsBody('oneDaySYSTEMMetrics', namespace, SCOPES.SYSTEM, 'now-24h', 'now', extraTags);
+        let overAllSYSTEMMetricsBody = getProfileMetricsBody('overAllSYSTEMMetrics', namespace, SCOPES.SYSTEM, 0, 0, extraTags);
         MyMetricApi
           .query(null, {...oneDayUSERMetricsBody, ...overAllUSERMetricsBody, ...oneDaySYSTEMMetricsBody, ...overAllSYSTEMMetricsBody})
           .subscribe(metrics => {
             let profilesToMetricsMap = {};
             Object.keys(metrics).forEach(query => {
               // oneDayMetrics, overMetrics are the keys for metrics for each profile.
-              let profileScope = query.indexOf('USER') !== -1 ? 'user' : 'system';
+              let profileScope = query.indexOf(SCOPES.USER) !== -1 ? SCOPES.USER : SCOPES.SYSTEM;
```

Comment:
does upper case vs lower case not matter in here?

Reply:
It shouldn't . As far as I tested this it matter. UI was just not consistent in passing the profile scope to backend.